ironfish/src/migrations/data/027-account-created-at-block.ts

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I recommend getting a second set of eyes on this migration. LGTM, but the migration I wrote didn't go well, so it's worth getting more scrutiny.